### PR TITLE
Added Contains, NotContains method in FrozenSet

### PR DESCRIPTION
**Pull request by ASP1234**
**Changes**

- Added contains, not Contains method in FrozenSet

- Added tests for the same

**Testing**

- Ant build successful

-  test_frozenSet successful

### CODE REVIEW

File: tests/datatypes/test_frozenset.py [hunk 1]
```diff
@@ -37,6 +37,60 @@ def test_creation(self):
...
+        self.assertCodeExecution("""
+            x = frozenset()
+            print(x)
+
+            x = frozenset({'a'})
+            print('a' in x)
+
+            y = {1,2}
+            x = frozenset(y)
+            print(2 in x)
```
Comment (cflee) [inline tests/datatypes/test_frozenset.py]: For each case, could there be something that tests both true/false cases, just to make sure they're working correctly? So for this one, perhaps `2` and `3`, or something like that.

(Also, is there much difference from the 'a' case above?)

File: tests/datatypes/test_frozenset.py [hunk 2]
```diff
@@ -37,6 +37,60 @@ def test_creation(self):
 
             """)
 
+    def test_contains(self):
+        # Normal Contains
+        self.assertCodeExecution("""
+            x = frozenset()
+            print(x)
```
Comment (cflee) [inline tests/datatypes/test_frozenset.py]: I'm not sure what this is testing, the repr should be tested elsewhere. Should this be ensuring that anything `in x` will be false?

File: tests/datatypes/test_frozenset.py [hunk 3]
```diff
@@ -37,6 +37,60 @@ def test_creation(self):
...
+
+            y = ['c']
+            x = frozenset(y)
+            print('c' in x)
+
+            y = (1,2)
+            x = frozenset(y)
+            print(3 in x)
+
+            y ='a'
```
Comment (ASP1234) [inline tests/datatypes/test_frozenset.py]: Done 👍

File: tests/datatypes/test_frozenset.py (hunk 1 above)
Comment (ASP1234) [inline tests/datatypes/test_frozenset.py]: Done 👍

File: tests/datatypes/test_frozenset.py (hunk 2 above)
Comment (ASP1234) [inline tests/datatypes/test_frozenset.py]: Changed to print(1 in x) to ensure nothing is in x. 😄

**Comment by cflee** [comment]
Thanks! :sparkles:

**Pull request merged**